File: src/Master/conexion.cpp
#include "conexion.h"

Conexion::Conexion(int input [2], int output [2])
{
    this->input[0]=input[0];
    this->input[1]=input[1];
    this->output[0]=output[0];
    this->output[1]=output[1];
    done=false;
}

File: src_python/Master/Audio/Conexion.py
import numpy as np

#Cada una de la conexiones se compone de un vector input y un vector output.
#Los valores de cada vector representan [idModulo, Numero de entrada/salida]
#Las salidas y entradas del sistema se conectan mediante el idModulo=-1
class Conexion :
    def __init__ (self, input , output) :
        self.input=input
        self.output=output
        self.done=0